### DISCUSSION TITLE
Is there a way to detach the tracer?

### DISCUSSION THREAD

**Discussion by Patrick-van-Halm** [vfsfitvnm/frida-il2cpp-bridge#339] [Q&A]
I would like to follow the code from a certain action so then in an implementation i would attach. But after a while I would like to detach the tracer is there anyway to do so?

**Comment by vfsfitvnm** [comment]
Not really, but if you really want to, you could call `Interceptor.detachAll()`.

The tracing mechanism is designed to be used alone, would you explain your user case further?
